client/my-sites/current-site/notice.jsx

```diff
@@ -84,6 +86,25 @@ export class SiteNotice extends React.Component {
 		const eventProperties = { cta_name: 'current_site_domain_notice' };
 		const { translate } = this.props;
 
+		if (
+			abtest( 'sidebarUpsellNudgeUnification' ) === 'variantShowUnifiedUpsells' &&
+			config.isEnabled( 'upsell/nudge-component' )
```

Comment:
Optional but I think we can remove the config flag here unless we were planning for refactors in other areas. If we think this might be off with the ab test on in some environments, I'd recommend checking for the flag first so we don't set a variant on a user.

Reply:
> If we think this might be off with the ab test on in some environments, I'd recommend checking for the flag first so we don't set a variant on a user.

In that case, we should also add the flag check to the JITMs for consistency.

Reply:
I think we can remove them, I realized after I added it that it was probably overkill, and that we'd only need the feature flag for applying UpsellNudge to other areas behind the scenes.